testkit/src/main/scala/org/bitcoins/testkit/node/NodeUnitTest.scala

```diff
@@ -60,6 +71,99 @@ trait NodeUnitTest extends BitcoinSFixture with EmbeddedPg {
 
   lazy val bitcoindPeerF = startedBitcoindF.map(NodeTestUtil.getBitcoindPeer)
 
+  val dummyChainApi: ChainApi = new ChainApi {
```

Comment:
Maybe could be called the `genesisChainApi`? This might be useful else where. It seems to me all of the information in this chainApi would work with a blockchain on the genesis block?